src/sbmlutils/report/sbmlinfo.py

```diff
@@ -156,6 +156,41 @@ def info_sbase(sbase: libsbml.SBase) -> Dict[str, Any]:
                     replaced_elements_combined = "".join(replaced_elements)
                 info["replaced_elements"] = replaced_elements_combined
 
+        # distrib
+        sbml_distrib = sbase.getPlugin("distrib")
+        if sbml_distrib and type(sbml_distrib) == libsbml.DistribSBasePlugin:
+
+            # list to store uncertainities
+            info["uncertainties"] = []
+
+            listOfUncertainties = sbml_distrib.getListOfUncertainties()
+            for uncertainty in listOfUncertainties:
+
+                u_dict = SBMLModelInfo.info_sbase(uncertainty)
+                u_dict["uncert_params"] = []
+
+                # uncertainty parameters defined for the uncertainty
+                uncertainty_params = uncertainty.getListOfUncertParameters()
```

Comment:
Same here, directly iterate over the list without generating another local variable.